.github/workflows/scripts/before_script.sh

```diff
@@ -10,9 +10,9 @@
 # make sure this script runs at the repo root
 cd "$(dirname "$(realpath -e "$0")")"/../../..
 
-set -euv
-
+set -mveuo pipefail
 source .github/workflows/scripts/utils.sh
+cmd_prefix bash -c "django-admin compilemessages"
```

Comment:
You can't edit this file directly, you need to create a new file called `.github/workflows/scripts/post_before_script.sh` and place the script you need there.

This file is managed by pulp plugin template and will be overriden.

Reply:
sorry, I misread the last comment, fixed it